framework/src/task_manager/task_manager_internal.h

```diff
@@ -72,6 +72,7 @@
 /* Wrapper of allocation APIs */
 #define TM_ALLOC(a)  malloc(a)
 #define TM_FREE(a)   free(a)
+#define TM_ASPRINTF(p, f, ...) asprintf(p, f, ##__VA_ARGS__)
```

Comment:
When I see debug.h, there is a conditional, CONFIG_CPP_HAVE_VARARGS. Don't you need to check that conditional here?